std/typecons.d

```diff
@@ -9035,3 +9035,220 @@ unittest
     assert((a ^ true) == Ternary.no);
     assert((a ^ false) == Ternary.yes);
 }
+
+/**
+Provides untyped storage of size `Size` aligned to `Alignment`.
+
+`Alignment` must be a power of 2.
```

Comment:
It should be a template constraint then.

Reply:
In this case it will cause compilation error anyway, and the compiler error message is more clear than the one from a constraint:
```
Error: alignment must be an integer positive power of 2, not 3LU
Error: template instance `std.typecons.AlignedStorage!(1LU, 3LU)` error instantiating
vs
Error: template instance `std.typecons.AlignedStorage!(1, 3)` does not match template declaration AlignedStorage(ulong Size, ulong Alignment) if (isPowerOf2(Alignment))
```